build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -131,8 +131,8 @@ Iterable<BuildAction> _createBuildActionsForBuilderInCycle(
       .where(builderApplication.filter)
       .map((p) => new BuildAction(b(options), p.name,
           builderOptions: options,
-          inputs: builderApplication.inputs,
-          excludes: builderApplication.excludes,
+          include: builderApplication.inputs,
```

Comment:
we should probably also update the names of the fields on `builderApplication`

Reply:
https://github.com/dart-lang/build/issues/726